packages/core/useVibrate/index.md

```diff
@@ -0,0 +1,81 @@
+---
+category: Sensors
+---
+
+# useVibrate
+
+Reactive hook to provide physical feedback with device vibration hardware using the [Vibration API](https://developer.mozilla.org/en-US/docs/Web/API/Vibration_API).
+
+## Usage
+
+```js
+import { useVibrate } from '@vueuse/core'
+
+const vibrating = ref(false)
+
+useVibrate(vibrating)
+
+// vibrating.value = true => vibration will start
+// vibrating.value = false => vibration will end
+
+```
+
+To start and stop the vibration using controls
+
+```js
+import { useVibrate } from '@vueuse/core'
+
+const vibrating = ref(false)
+
+const { pause, resume } = useVibrate(vibrating, {
+  pattern: [300, 100, 200, 100, 1000, 300],
+  loop: true,
+})
+
+// resume() => vibration will start
+// pause() => vibration will end
```

Comment:
For me is not clear what differentness between using `vibrating.value = true/false` and using `pause/resume` controls.

Reply:
This is probably something we should discuss in #518 as well so we can make recommendations on when to use play/pause vs controlling a value like `vibrating`